app/views/users/index.html.erb

```diff
@@ -1,4 +1,3 @@
-<%= stylesheet_link_tag('index') %>
 <div class="container">
   <div class="user">
   <img src="<%= image_path('example.com.png') %>" alt="image">
```

Comment:
- [ ] `Required`
- [ ] Amazing so far! 👏 
- [ ] However there is an Error with The asset "example.com.png" is not present in the asset pipeline.
- [ ] Added a screenshot below
- [ ] Please fix this. 🥂 

![image](https://github.com/firstpj/blog-app/assets/66230306/907ded4d-eda1-42ec-a8c0-85410d55b958)